annotation-file-utilities/build.gradle

```diff
@@ -44,17 +45,7 @@ configurations {
 }
 
 dependencies {
-  if (isJava8) {
-    ext.errorproneJavacVersion = "9+181-r4173-1"
-    javacJar group: 'com.google.errorprone', name: 'javac', version: "$errorproneJavacVersion"
-    implementation "com.google.errorprone:javac:$errorproneJavacVersion"
-    errorproneJavac("com.google.errorprone:javac:$errorproneJavacVersion")
-  }
-  if (JavaVersion.current() == JavaVersion.VERSION_1_8) {
-    implementation 'org.plumelib:options:1.0.6'
-  } else {
-    implementation 'org.plumelib:options:2.0.3'
-  }
+  implementation 'org.plumelib:options:2.0.3'
```

Comment:
I think Options needs two artifacts: one that is required to build the project and one that is only used by `updateUserOptions`.  Only the second jar file will need to be different if JDK 8 is used when running the `updateUserOptions`.